.k8s.mk

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+BUILD_CONTAINERS=nsc nsmd nsmdp nsmd-k8s icmp-responder-nse vppagent-dataplane devenv crossconnect-monitor
```

Comment:
Are we sure these are all we build? We lack at least `vppagent-icmp-responder-nse`. And then maybe the firewall.